webrtc/idlharness.https.window.js

```diff
@@ -65,13 +65,18 @@ function asyncInitTransports() {
 
     const dtlsTransport = sctpTransport.transport;
     assert_true(dtlsTransport instanceof RTCDtlsTransport,
-      'Expect sctpTransport.transport to be instance of RTCDtlsTransport');
+      'Expect dtlsTransport.transport to be instance of RTCDtlsTransport');
     idlTestObjects.dtlsTransport = dtlsTransport;
 
     const iceTransport = dtlsTransport.iceTransport;
     assert_true(iceTransport instanceof RTCIceTransport,
-      'Expect sctpTransport.transport to be instance of RTCDtlsTransport');
+      'Expect iceTransport.transport to be instance of RTCIceTransport');
     idlTestObjects.iceTransport = iceTransport;
+
+    const iceCandidatePair = iceTransport.getSelectedCandidatePair();
+    assert_true(iceCandidatePair instanceof RTCIceCandidatePair,
```

Comment:
For development, you should test that you get here and get a candidate pair - assert_not_null(iceCandidatePair).
 think the test will only work if the candidate is selected before you get this far - which is timing dependent, but can't be depended on without waiting on a state change.

Reply:
indeed; amended the test to trigger and await the proper ice connection state